docs/manual/kinds/cjunosevolved.md

```diff
@@ -27,24 +27,39 @@ The container image can be freely downloaded from the [Juniper support portal](h
 !!!note
     Containers with cJunosEvolved inside will take ~5min to boot to login prompt.
     You can monitor the progress with `docker logs -f <container-name>`.
+    Alternatively, you can also use the `docker exec -ti <container-name> cli` as shown in more detail below.
+
 !!!note
-    The management port IP is assigned by containerlab and is merged into the cJunosEvolved startup config.
-    Please refer to `docs/lab-examples/srl-cjunosevolved.md` for an example.
+    The management port IP is assigned by containerlab and is merged into the startup config by the cJunosEvolved container. The default startup config allows for SSH access with the default credentials below.
 
 === "CLI via SSH"
```

Comment:
@vista- can you also switch to block style syntax?

```
/// note
text goes here
///
```

and for tabs:

```
/// tab | some name
///
/// tab | another tab
///
```

Reply:
Will do this morning